loops/2_loops_advance.php

```diff
@@ -8,3 +8,17 @@
  *
  * Loop should print() from 0 to $limit
  */
+
+
+/**
+ * Recursive function counting from offset to limit
+ * limit (number)  - the boundry limit of iteration
+ * offset (number) - value function will count from
+ */
+function recursiveLoop($limit, $offset=0) {
+    if ($offset < $limit) {
+        print($offset);
+        recursiveLoop($limit, $offset + 1);
+    }
+}
+recursiveLoop(10);
```

Comment:
`No newline at end of file`.

Pamiętaj o tym, aby zawsze ostatni wiersz był pusty.
Jeśli nie chcesz o tym pamiętać, zainstaluj plugin do IDE, który będzie pamiętał za Ciebie.